### ISSUE TITLE
Filter Control: onPageChange events triggered a lot on reload using cookies

### ISSUE THREAD

**Issue by francogp** [wenzhixin/bootstrap-table#6270]
### Bootstraptable version(s) affected

1.20.2

### Description

Every input saved on a cookie, trigger an onPageChange event when page is reloaded or when hit RUN on the example. It should only be triggered once.

### Example(s)

type this filter on the example

![image](https://user-images.githubusercontent.com/2739706/179098839-d4a20a7a-caef-4fe7-81a1-4e4d21c719b7.png)

Then hit RUN, and you will see the event execute 4 times. On a real web, when I reload the page, it also executes 4 time the event.

https://live.bootstrap-table.com/code/francogp/12103

**Comment by UtechtDustin** [comment]
This happens because the table currently will be rendered multiple times (through the historical growth of BT).
In that case the `filter-control` extension will call the internal function `onSearch` per input which will call `updatePagination` and this function will trigger the events.
I don't think we can fix that right now, for that we have to rewrite a LOT of the BT.
That should be goal for a further version, but this could take a long time from now.

**Comment by djhvscf** [comment]
Yes, @UtechtDustin said.. this is a known issue and also a difficult issue to solve since it is related to how the core works. At the moment there's no any workaround for this problem.
